### PR TITLE
chore(release): merge `next` into `main`

### PR DESCRIPTION
**Pull request by sherwinski**
This to namely release changes introduced in a4884abdfc43684acb58b8a0711f52b3986f459e more generally.
It's also to test if semantic-release will correctly handle merging an unstable branch into the default one.

### CODE REVIEW

File: src/validators.js [hunk 1]
```diff
@@ -37,7 +39,9 @@ function isJSONString(body) {
 
 function isJSONObject(body) {
   try {
-    assert.strictEqual(typeof body, 'object');
+    if (typeof body !== 'object' || Array.isArray(body) || body === null) {
```
Comment (gmp) [inline src/validators.js:42]: This is technically doing more conditional checks than the original. Just calling that out in case there's anything there. Looks like you added some tests for this though.

**Review by gmp — APPROVED** [review]

File: src/validators.js (hunk 1 above)
Comment (sherwinski) [reply src/validators.js:42]: Yeah it was added to catch an edge case. So I guess in a sense this is technically a bug fix?
